components/user_settings/import_theme_modal.tsx

```diff
@@ -15,7 +15,7 @@ const ActionTypes = Constants.ActionTypes;
 
 type State = {
     value: string;
-    inputError: any;
+    inputError: JSX.Element | null;
```

Comment:
Can use the `React.ReactNode` type here instead, it covers all types that can be rendered to the DOM by React.